### DISCUSSION TITLE
get_prompt function failed in langfuse-python 2.28.3

### DISCUSSION THREAD

**Discussion by nathan-vo810** [langfuse/langfuse#1979] [Support] [answered]
I'm currently on version 2.27.0 and when upgrading to 2.28.3, the get_prompt function failed. 

As I inspect in the code, the get_prompt function failed because url has been changed from `api/public/prompts` to `api/public/v2/prompts`.

How should I handle this problem?

**Comment by marcklingen** [comment, marked as answer]
do you self-host langfuse? If yes you just need to update to the latest container version. we make sure that langfuse container is backwards compatible with old sdk version but we assume that the application container is continuously upgraded
